code/src/reducers/todos.js

```diff
@@ -0,0 +1,63 @@
+import { createSlice } from '@reduxjs/toolkit';
+import moment from 'moment';
+
+export const todos = createSlice({
+  name: 'todos',
+  initialState: {
+    tasks: [
+      {id: 1, text: 'enjoy life', done: false, created: moment(new Date()).format()},
+      {id: 2, text: 'spread love', done: false, created: moment(new Date()).format()},
+      {id: 3, text: 'relax', done: false, created: moment(new Date()).format()}
+    ]
+  }, 
+  reducers: {
+    removeItem: (store, action) => {
+      const itemId = action.payload;
+      const filteredList = store.tasks.filter(item => item.id !== itemId);
+
+      store.tasks = filteredList
+    },
+    toggleDone: (store, action) => {
+      const itemId = action.payload;
+      const currentItem = store.tasks.find(item => item.id === itemId);
+      
+      currentItem.done === true ? currentItem.done = false : currentItem.done = true;
```

Comment:
That works but you could have used 
```suggestion
      currentItem.done === !currentItem.done;
```
to toggle the done status